### PR TITLE
Revert "Docker: Include Root CA cert needed for APNs (#141)"

### PR DESCRIPTION
**Pull request by csett86**
This reverts commit d477292a9ce5ff885c58f781b9652a42597a31ac.

Since March 29 2021 APNs is no longer using GeoTrust, so this can be removed.
See https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server

### CODE REVIEW

**Review by anoadragon453 — CHANGES_REQUESTED** [review]
From the link:

> Communication between your provider server and APNs must take place over a secure connection. Creating that connection requires installing the GeoTrust Global CA root certificate (until March 29, 2021) and the AAA Certificate Services root certificate (starting March 29, 2021) on each of your provider servers.

That date was indeed now passed, and buster's `ca-certificates` package [looks to contain](https://packages.debian.org/buster/all/ca-certificates/filelist) `Comodo_AAA_Services_root.crt`.

This all lines up with the information in #173, so I think we're good here. Thanks for taking the time to put together this PR!

Last thing I need from you is to [sign off](https://github.com/matrix-org/sygnal/blob/881ab216ec5ff949c388109e659765265ee7358f/CONTRIBUTING.md#sign-off) on your changes :slightly_smiling_face:

**Comment by csett86** [comment]
Sure @anoadragon453, should have added that in the initial PR comment:

Signed-off-by: Christoph Settgast <csett86@web.de>

File: changelog.d/208.docker [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove manually added GeoTrust from docker image as Apple is no longer using it.
```
Comment (anoadragon453) [inline changelog.d/208.docker:1]:
```suggestion
Remove manually added GeoTrust Root CA certificate from docker image as Apple is no longer using it.
```

**Review by anoadragon453 — APPROVED** [review]
Thank you!

**Pull request merged**